app/controllers/api/places_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Api
+  class PlacesController < ApplicationController
+    before_action :set_place, only: %i[show update destroy]
+
+    def index
+      @places = Place.all
+      render json: @places
+    end
+
+    def show
+      render json: @place
+    end
+
+    def create
+      @place = Place.new(place_params)
+      if @place.save
+        render json: @place, status: :created
+      else
+        render json: @place.errors, status: :unprocessable_entity
+      end
+    end
+
+    def update
+      if @place.update(place_params)
+        render json: @place, status: :ok
+      else
+        render json: @place.errors, status: :unprocessable_entity
+      end
+    end
+
+    def destroy
+      @place.destroy
+      head :no_content
+    end
+
+    private
+
+    def set_place
+      @place = Place.find(params[:id])
```

Comment:
`params[:id]` should be permitted.
update `place_params` and work only with them